python/lsst/ip/isr/isrTask.py

```diff
@@ -46,18 +46,35 @@
 from . import linearize
 
 from .assembleCcdTask import AssembleCcdTask
-from .crosstalk import CrosstalkTask
+from .crosstalk import CrosstalkTask, CrosstalkCalib
 from .fringe import FringeTask
 from .isr import maskNans
 from .masking import MaskingTask
 from .overscan import OverscanCorrectionTask
 from .straylight import StrayLightTask
 from .vignette import VignetteTask
+from lsst.daf.butler import DataCoordinate, DimensionGraph, DimensionUniverse
 
 
 __all__ = ["IsrTask", "IsrTaskConfig", "RunIsrTask", "RunIsrConfig"]
 
 
+def ctSourceLUF(datasetType, registry, quantumDataId, collections):
```

Comment:
Docstring please, and consider renaming the function with fewer acronyms (I can guess ct is crosstalk but am not sure what LUF is) ... unless ct is connectionType which has been imported as cT ... why